modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
I can put a hard coded check, until now I only encountered 3 codes for dead trees..I could also look for "dead" in the commonname but there could be trees that have "dead" in their common name